--- init/src/test/java/com/larpconnect/njall/init/GuiceVerticleFactoryTest.java
+++ init/src/test/java/com/larpconnect/njall/init/GuiceVerticleFactoryTest.java
@@ -21,6 +21,7 @@
   static final class TestVerticle extends AbstractVerticle {}

   @Test
+  @SuppressWarnings("deprecation")
   public void createVerticle_validClass_success(VertxTestContext testContext) {
     Injector injector =
         Guice.createInjector(
@@ -55,6 +56,7 @@
   }

   @Test
+  @SuppressWarnings("deprecation")
   public void createVerticle_missingClass_failure(VertxTestContext testContext) {
     Injector injector = Guice.createInjector(new com.larpconnect.njall.common.CommonModule());
     var factory = new GuiceVerticleFactory(injector);
