--- init/src/main/java/com/larpconnect/njall/init/VerticleLifecycle.java
+++ init/src/main/java/com/larpconnect/njall/init/VerticleLifecycle.java
@@ -19,7 +19,7 @@
 import org.slf4j.LoggerFactory;

 final class VerticleLifecycle extends AbstractIdleService implements VerticleService {
-  private static final int CONFIG_LOAD_TIMEOUT_SECONDS = 5;
+  private static final int CONFIG_LOAD_TIMEOUT_SECONDS = 30;
   private final Logger logger = LoggerFactory.getLogger(VerticleLifecycle.class);

   private final ImmutableList<Module> modules;
